ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/content/granite/field/basename/.content.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this will add a field but it's nowhere included in a dialog

Reply:
> this will add a field but it's nowhere included in a dialog
[
](https://github.com/orbinson/aem-dictionary-translator/blob/main/ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/content/granite/dialog/dictionary/language/create/.content.xml)